### PR TITLE
Document workaround for conflicting Thymeleaf dependency (spring-clou…

### PR DESCRIPTION
**Pull request by stovocor**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@stovocor Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=3291)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-netflix&pullRequestId=3291) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: docs/src/main/asciidoc/spring-cloud-netflix.adoc [hunk 1]
```diff
@@ -335,6 +335,16 @@ This section describes how to set up a Eureka server.
 To include Eureka Server in your project, use the starter with a group ID of `org.springframework.cloud` and an artifact ID of `spring-cloud-starter-netflix-eureka-server`.
 See the http://projects.spring.io/spring-cloud/[Spring Cloud Project page] for details on setting up your build system with the current Spring Cloud Release Train.
 
+If your project already uses Thymeleaf as its template engine, the Freemarker templates of the Eureka server may not be loaded correctly. In this case it is necessary to configure the template loader manually:
```
Comment (spencergibb) [inline docs/src/main/asciidoc/spring-cloud-netflix.adoc]: Add `NOTE:`

**Comment by ryanjbaxter** [comment]
Could you submit this against the 2.0.x branch?

**Comment by stovocor** [comment]
The new base is now 2.0.x

**Pull request merged**